contracts/TwabRewards.sol

```diff
@@ -108,21 +125,23 @@ contract TwabRewards is ITwabRewards {
         uint256 _nextPromotionId = _latestPromotionId + 1;
         _latestPromotionId = _nextPromotionId;
 
+        uint256 _amount;
+
+        unchecked {
+            _amount = _tokensPerEpoch * _numberOfEpochs;
+        }
+
         _promotions[_nextPromotionId] = Promotion(
```

Comment:
Would be safer to use named variables here.  Just good practice!

i.e.

```
 = Promotion({
startTimestamp: _startTimestamp
// etc
})
```

Reply:
Done: https://github.com/pooltogether/v4-periphery/pull/48/commits/86e130f68e5beac4fa7078fa560d475cfb679582